sort_names.py

```diff
@@ -32,45 +34,90 @@ def get_short_list(filename, first_names, last_names):
             file, delimiter=","
         )  # delimiter="," - if the data is concatenated in the first column by ','
         # delimiter=";" - if the data in the different column
+        first_row = next(csvreader)
         read_file = list(csvreader)
-        num_people = len(read_file) - 1
+        num_people = len(read_file)
+        count_name = count_last = count_pos = count_age_race = num_people
         print("--- START ---")
-        print("Estimated time ~ {} seconds".format(num_people * 1.2))
+        print("Estimated time ~ {} minutes".format(round(num_people * 1.2 / 60)))
+        writer.writerow(first_row)
 
         for row in read_file:
-            if (
-                (row[24] not in first_names)
-                and (row[25] not in last_names)
-                and (forbidden_word not in row[43])
-                and face_filter(row[30], limit_age, races)
-            ):
-                writer.writerow(row)
-                count += 1
+            if name_filter(row[24], first_names):
```

Comment:
I would extract conditions in the separate function from the main flow for clarify.

Reply:
It wasn't easy, I have some trouble with counts, but here is result)